modules/apps/change-tracking/change-tracking-web/src/main/resources/META-INF/resources/publications/js/views/ChangeTrackingChangeView.js

```diff
@@ -345,6 +345,14 @@ export default function ChangeTrackingChangeView({
 
 	const getDiscardURL = useCallback(
 		(node) => {
+			if (
+				!discardURL ||
+				discardURL === null ||
+				discardURL === undefined
+			) {
```

Comment:
These checks `discardURL === null || discardURL === undefined` seem redundant. I believe `!discardURL` should catch both null and undefined. Is the check not working with just `if(!discardURL)`?

Reply:
My bad, it is because I'm very inexperienced with JavaScript, so I had to keep trying different things until it worked. It does work with just `if(!discardURL)`, so I've edited the commit and force-pushed it.